components/calendar/src/astronomy.rs

```diff
@@ -1180,13 +1183,12 @@ impl Astronomical {
     ///
```

Comment:
nit: clarify what the new `lunar_phase` parameter is in the docs

Reply:
done.